data/odyssey_functions.csv

```diff
@@ -53460,6 +53460,7 @@ Address,Quality,Size,Name
 0x000000710088416c,U,001232,_ZN2al18EffectShaderHolderC2EPNS_10PtclSystemEPN3agl11DrawContextEPNS_14EffectEnvParamE
 0x000000710088463c,U,000072,
 0x0000007100884684,U,000040,
+0x00000071008846ac,U,000140,
```

Comment:
Is there a reason this function here wasn't there before? I assume this could be a difference in IDA (perhaps versions?) deciding to create a function while a different analysis didn't.

Reply:
Looking at my IDA repo, there's obviously a function there while IDA doesn't define one. It looks like it's a lambda type thing with the function being stored in a class, and possibly that just confused IDA. LGTM

Reply:
I don't know why it didn't get recognized the first time around, but I also don't remember if I actually manually fixed the function to exist... however, between originally generating the CSV and now updating it from my current IDA, a lot of time has passed, so maybe I did change it at some point, but just don't remember? Either way, as you said, it's referenced in the constructor in a way that it can only mean that a function exists here (not a reference within the same function/...), so I guess that this is correct now.